settings.ui

```diff
@@ -143,6 +143,13 @@
          </property>
         </spacer>
        </item>
+       <item>
+        <widget class="QCheckBox" name="searchForUpdates">
+         <property name="text">
+          <string>Search for updates (Windows Only)</string>
```

Comment:
`Only` &rarr; `only`

I think labeling the button with OS specific stuff is not a good idea.
I'd rather have it hidden / disabled on other platforms with a small note.